source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -200,6 +200,11 @@ address
 
 The hostname or IP address, and port number, of a MongoDB server.
 
+network error
+`````````````
+
+An error that occurs while reading from or writing to a network socket.
```

Comment:
Do you want to clarify that this is "excluding a timeout"?

Reply:
We could add it but at least in the section where the ambiguity exists, network errors and network timeouts are separate, at least that's how I read it.

> If there is a *network error or timeout* on the connection before the handshake completes

> If there is a *network timeout* on the connection after the handshake completes

Then again, a following sentence refers to a network error as "other network error"

> If, however, there is some *other network error* on the connection after the handshake completes

But I can add that clarification if we think it'll help.

Reply:
Maybe leave this definition as is and add a definition for "network timeout" as being either a network error indicating a socket timeout like ETIMEDOUT or a user-space timeout like with `select`?

Reply:
Would "A timeout that occurs while reading from or writing to a network socket" work? It sounds like we need to have a larger discussion around what a "network timeout" is. Not sure if that's in scope for this change though.

Reply:
Added a definition for network timeout.